packages/web-worker/src/index.ts

```diff
@@ -1,3 +1,4 @@
-export {createWorker} from './create';
+export {createWorker, terminate} from './create';
 export {retain, release} from './memory';
 export {SafeWorkerArgument} from './types';
+export {expose} from './worker';
```

Comment:
I noticed this wasn't exposed, should it be?

Reply:
No, the intention is just to leave it where it is, it's available from a pseudo-private `@shopify/web-worker/worker` entry that is used internally by the tools.

Reply:
However, we should be exporting the function of the same name from `./create`